schedule/yam/agama_auto.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: agama_auto
+description: >
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation for autoinstallation with agama.
```

Comment:
we can remove "for autoinstallation" in this sentence.

Reply:
Updated.